### PR TITLE
Modify build.sh to skip native building

### PR DESCRIPTION
**Pull request by iamjasonp**

### CODE REVIEW

**Comment by iamjasonp** [comment]
@StephenBonikowsky - I think this is what's causing the build failures we are seeing in CI

File: build.sh [hunk 1]
```diff
@@ -461,8 +461,8 @@ if $__buildnative; then
     prepare_native_build
 
     # Build the corefx native components.
-
-    build_native_corefx
+    # WCF does not have a native component
+    # build_native_corefx
```
Comment (roncain) [inline build.sh]: LGTM

Nit: I believe @StephenBonikowsky is using this comment convention to make these easier to spot:
```
  # *** start WCF Content ***
```

Comment (iamjasonp) [inline build.sh]: Done.

**Comment by iamjasonp** [comment]
I took this opportunity to fix up the tabs/spaces issues as well. 

@StephenBonikowsky, you might want to double check your text editor settings to insert tabs as spaces :smile:

**Comment by iamjasonp** [comment]
... also, @StephenBonikowsky, if you approve of this change, can you LGTM and merge? I need to head off shortly. Thanks!

**Comment by StephenBonikowsky** [comment]
LGTM

**Pull request merged**